src/meta/src/hummock/manager/mod.rs

```diff
@@ -2532,6 +2586,7 @@ impl HummockManager {
                     }
                 }
 
+                table_vnode_partition_mapping.insert(*table_id, partition_vnode_count);
```

Comment:
vnode parition mapping set in L2553 can be overwritten here. Is this expected?

Reply:
Okay, I'm going to remove it, currently we'll set it up when we build compact_task